convert_models/run.sh

```diff
@@ -1,6 +1,7 @@
 #! /bin/bash
 
 hf_model_url=${HF_MODEL_URL}
+hf_token=${HF_TOKEN}
```

Comment:
```suggestion
hf_token=${HF_TOKEN:="None"}
```
This should overcome the issue of when its not needed. 

Can you also add this token parameter as a textbox in the UI? otherwise UI users won't be able to access token gated models.

Reply:
Follwing your comments, I Modified the run.sh, ui.py and README.md. Please take a look.

Here is my testing:
![Screenshot at 2024-08-14 17-35-34](https://github.com/user-attachments/assets/9adf63da-5419-4789-813c-b7b2c0e246e3)